### ISSUE TITLE
status.version does not return server version

### ISSUE THREAD

**Issue by nico-qwer** [py-mine/mcstatus#451]
I don't know if that's intended or not, but when i do this:
```
status = server.status()
print(status.version)
```
i get this in the console:
```
<mcstatus.pinger.PingResponse.Version object at 0x00000211FD2BA710>
```
I would like to get 1.19.2 instead, as this is the version of my minecraft server.
Is there a way to do this?

edit: typo

**Comment by kevinkjt2000** [comment]
Looks like `Version` is missing a `__str__` method.

**Comment by nico-qwer** [comment]
Thanks, that did it!

**Comment by nico-qwer** [comment]
Thanks that did it !

**Comment by kevinkjt2000** [comment]
Keeping this open since I was counting the missing `__str__` method as a bug. #306 rewrites the `Version` class, so this issue can be closed after it has been verified that the new version classes have a `__str__` method for printing.

**Comment by PerchunPak** [comment]
Currently, `__str__` in #306 gives just
```py
>>> from mcstatus.status_response import JavaStatusVersion
>>> v = JavaStatusVersion("1.19.2", 123)
>>> v
JavaStatusVersion(name='1.19.2', protocol=123)
>>> str(v)
"JavaStatusVersion(name='1.19.2', protocol=123)"
```
Should it instead give `1.19.2`? I'm quite unsure. `__str__` can be used anywhere, and this *anywhere* can produce unexpected results and possible bugs.

**Comment by ItsDrike** [comment]
Yeah, I agree that we probably shouldn't make `__str__` just return the pure minecraft version, as the class also includes the protocol version (and perhaps other things in the future). I don't see anything wrong with having the users access the actual underlying variable with `version.name`

**Comment by kevinkjt2000** [comment]
Fixed by #306
